src/treeview_commands.ml

```diff
@@ -43,7 +43,31 @@ let terminal_item =
   Vscode.TreeItem.set_command item command;
   item
 
-let items = [ select_sandbox_item; terminal_item ]
+
+  let construct_item =
+    let icon =
+      `LightDark
+        Vscode.TreeItem.LightDarkIcon.
+          { light = `String (Path.asset "collection-light.svg" |> Path.to_string)
+          ; dark = `String (Path.asset "collection-dark.svg" |> Path.to_string)
+          }
+    in
+    let label =
+      `TreeItemLabel
+        (Vscode.TreeItemLabel.create ~label:"Construct" ())
```

Comment:
We should provide a better description as a command name, something like "List values that can fill the selected typed-hole`. This should be done in other places too.